test/fixtures/poro.rb

```diff
@@ -248,6 +259,22 @@ def maker
   has_one :imageable, serializer: PolymorphicHasManySerializer, polymorphic: true
 end
 
+PolymorphicSimpleSerializer = Class.new(ActiveModel::Serializer) do
+  attributes :id
+end
```

Comment:
simple serializer to demonstrate polymorph inclusions (as they didnt have their own)